zkevm-circuits/src/lib.rs

```diff
@@ -34,6 +34,7 @@ pub mod mpt_circuit;
 pub mod pi_circuit;
 #[cfg(not(target_arch = "wasm32"))]
 pub mod root_circuit;
+pub mod sig_circuit;
```

Comment:
We should include ecc_circuit here too.
```suggestion
pub mod sig_circuit;
pub mod ecc_circuit;
```

Reply:
fixed in [5cb6a0e](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1720/commits/5cb6a0e7f7c799bc51f9de8645b83f548174320b)

Reply:
I see the reason not to include ecc_circuit module in this PR now. The status quo makes sense to me.